src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,50 +1,91 @@
-import { useState } from 'react';
+import { FormEvent, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = { onAdd: (movie: Movie) => void };
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
make one state as object for all input fields and use event.target.name for change proper field. attribute name in input should be the same to inputs state appropriate key
```
```
```suggestion
  const [newMovie, setNewMovie] = useState({
    title: '',
    ...
  });
```